fineract-core/src/main/java/org/apache/fineract/portfolio/tax/request/TaxComponentRequest.java

```diff
@@ -38,7 +38,7 @@ public class TaxComponentRequest implements Serializable {
     private String name;
     private BigDecimal percentage;
     private Integer debitAccountType;
-    private Long debitAccountId;
+    private Long debitAcountId;
```

Comment:
typo...

Reply:
it is not a type, intentional because this is the spelling used in many places including web app.

Reply:
@sidhantgoel Please undo this!